frontend/config-core_demo.json

```diff
@@ -1,52 +1 @@
-{
-  "@openmrs/esm-patient-chart-app": {
-    "restrictByVisitLocationTag": true,
-    "showUpcomingAppointments": true,
```

Comment:
I guess you forgot to remove this from: https://github.com/openmrs/openmrs-distro-referenceapplication/blob/main/frontend/Dockerfile#L39

Reply:
Done!

Reply:
The reason it fails when the above line + file is removed because that config file is no longer in the html directory of the container.

The frontend configs from content packages are loaded to `distro/target/sdk-distro/web/openmrs_spa` directory from the sdk. We should somehow copy all the frontend config files to the html directory of the frontend container.